source/common/router/req_header_formatter.h

```diff
@@ -0,0 +1,97 @@
+#pragma once
+
+#include <cstdint>
+#include <iostream>
+#include <string>
+#include <unordered_map>
+
+#include "common/common/logger.h"
+#include "common/http/access_log/access_log_formatter.h"
```

Comment:
i guess it is required for Envoy::Http::AccessLog::AccessLogFormatUtils::protocolToString(..) method in req_header_formatter.cc